BitwardenShared/Core/Platform/Models/Enum/FeatureFlag.swift

```diff
@@ -5,12 +5,36 @@ import Foundation
 /// An enum to represent a feature flag sent by the server
 ///
 enum FeatureFlag: String, Codable {
+    /// Flag to enable/disable email verification during registration
+    /// This flag introduces a new flow for account creation
+    case emailVerification = "email-verification"
+
     /// A feature flag for the intro carousel flow.
     case nativeCarouselFlow = "native-carousel-flow"
 
     /// A feature flag for showing the unassigned items banner.
     case unassignedItemsBanner = "unassigned-items-banner"
-    /// Flag to enable/disable email verification during registration
-    /// This flag introduces a new flow for account creation
-    case emailVerification = "email-verification"
+
+    // MARK: Test Flags
+
+    /// A test feature flag that isn't remotely configured.
+    case testLocalFeatureFlag = "test-local-feature-flag"
+
+    /// A test feature flag that can be remotely configured.
+    case testRemoteFeatureFlag = "test-remote-feature-flag"
+
+    // MARK: Properties
+
+    /// Whether this feature can be enabled remotely.
+    var isRemotelyConfigured: Bool {
+        switch self {
+        case .emailVerification,
```

Comment:
❓Maybe I'm missing something but isn't **Email Verification** feature complete yet? cc: @andrebispo5

Reply:
Yes but I think it won't be released for now, that is why it's disabled here.